# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/dump/dump-devfs.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./compat/include/pathnames.h.orig	Mon Nov 20 13:30:38 2000
+++ ./compat/include/pathnames.h	Mon Nov 20 13:32:06 2000
@@ -49,7 +49,7 @@
 #endif
 
 #ifndef _PATH_DEFTAPE
-#define	_PATH_DEFTAPE	"/dev/st0"
+#define	_PATH_DEFTAPE	"/dev/tape/tape0"
 #endif
 
-#define	_PATH_RMT	"/etc/rmt"		/* path on remote host */
+#define	_PATH_RMT	"/sbin/rmt"		/* path on remote host */
